reference-implementations/icrc25/canister/dfx.json

```diff
@@ -0,0 +1,46 @@
+{
+  "canisters": {
+    "ledger": {
+      "type": "custom",
+      "candid": "https://raw.githubusercontent.com/dfinity/ic/a4191e757dbc581cbc8cef9160aea1e852218a1a/rs/rosetta-api/icrc1/ledger/ledger.did",
```

Comment:
I think this approach here is sufficient to reference a ledger canister. I would drop the `dfx.ic.json` together with the contents of the `icrc1_ledger` folder.